tests/app/celery/test_nightly_tasks.py

```diff
@@ -76,7 +76,7 @@ def test_will_remove_csv_files_for_jobs_older_than_seven_days(notify_db, notify_
     """
     Jobs older than seven days are deleted, but only two day's worth (two-day window)
     """
-    mocker.patch("app.celery.nightly_tasks.s3.remove_job_from_s3")
+    mocker.patch("app.celery.nightly_tasks.s3.remove_job_batch_from_s3")
```

Comment:
Oh I see you added `batch` in fact, instead of pluralizing it. 😅 oh humm alright

Reply:
all better now!